views/aggregate-flow-metadata.json

```diff
@@ -1,6 +1,17 @@
 {
     "prefect-airbyte": {
         "run-connection-sync": {
+            "description": {
+                "summary": "A flow that triggers a sync of an Airbyte connection and waits for it to complete.",
+                "returns": "`AirbyteSyncResult`: Model containing metadata for the `AirbyteSync`.",
```

Comment:
Should returns be a list as well?

Reply:
based on my convo with evan, this will likely not be used yet, so I don't feel the need to split hairs on this before its being used by anyone or know how it will be rendered